service/controller/app/v1/resource/chartoperator/create.go

```diff
@@ -0,0 +1,46 @@
+package chartoperator
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/helmclient"
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/app-operator/service/controller/app/v1/controllercontext"
+	"github.com/giantswarm/app-operator/service/controller/app/v1/key"
+)
+
+func (r Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
+	cr, err := key.ToCustomResource(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	cc, err := controllercontext.FromContext(ctx)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	// Check whether TC has chart-operator deployment yet.
+	{
+		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("finding chartoperator release %#q in tenant cluster", chartOperatorRelease))
+		_, err := cc.HelmClient.GetReleaseContent(ctx, chartOperatorRelease)
+		if helmclient.IsReleaseNotFound(err) {
+			r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("no chartoperator release %#q in tenant cluster", chartOperatorRelease))
```

Comment:
I would say let's either drop the log or align it with the one above so that it reads "did not find chart-opertor ...".

Reply:
+1 that is aligned with https://github.com/giantswarm/fmt/blob/master/go/log_messages.md#fact-gathering.